doc/user/caching.xml

```diff
@@ -559,8 +541,8 @@ Program('prog', ['f1.c', 'f2.c', 'f3.c', 'f4.c', 'f5.c'])
 
     <scons_example name="custom_caching">
       <file name="SConstruct" printme="1">
-import SCons
 import os
+import SCons.CacheDir
```

Comment:
Is this necessary and missing originall?

Reply:
Yes.  `CacheDir` the function is global to sconscripts, but not the class.
```py
scons: Reading SConscript files ...
NameError: name 'SCons' is not defined:
  File "/home/mats/github/scons/tmp/cdir/SConstruct", line 3:
    class CustomCacheDir(SCons.CacheDir.CacheDir):
```